src/mmw/js/src/compare/views.js

```diff
@@ -6,17 +6,151 @@ var _ = require('lodash'),
     App = require('../app'),
     coreModels = require('../core/models'),
     coreViews = require('../core/views'),
+    models = require('./models'),
     modelingModels = require('../modeling/models'),
     modelingViews = require('../modeling/views'),
     modelingControls = require('../modeling/controls'),
     modConfigUtils = require('../modeling/modificationConfigUtils'),
     compareWindowTmpl = require('./templates/compareWindow.html'),
+    compareWindow2Tmpl = require('./templates/compareWindow2.html'),
+    compareTabPanelTmpl = require('./templates/compareTabPanel.html'),
+    compareInputsTmpl = require('./templates/compareInputs.html'),
+    compareScenarioItemTmpl = require('./templates/compareScenarioItem.html'),
+    compareTableRowTmpl = require('./templates/compareTableRow.html'),
     compareScenariosTmpl = require('./templates/compareScenarios.html'),
     compareScenarioTmpl = require('./templates/compareScenario.html'),
     compareModelingTmpl = require('./templates/compareModeling.html'),
     compareModificationsTmpl = require('./templates/compareModifications.html'),
     synchronizer = modelingControls.PrecipitationSynchronizer;
 
+var CompareWindow2 = Marionette.LayoutView.extend({
+    template: compareWindow2Tmpl,
+
+    id: 'compare-new',
+
+    ui: {
+        closeButton: '.compare-close > button',
+    },
+
+    events: {
+        'click @ui.closeButton': 'closeView',
+    },
+
+    regions: {
+        tabRegion: '.compare-tabs',
+        inputsRegion: '.compare-inputs',
+        scenariosRegion: '#compare-title-row',
+        sectionsRegion: '.compare-sections',
+    },
+
+    onShow: function() {
+        this.tabRegion.show(new TabPanelsView({
+            collection: this.model.get('tabs'),
+        }));
+        this.inputsRegion.show(new InputsView({
+            model: this.model,
+        }));
+        this.scenariosRegion.show(new ScenariosRowView({
+            collection: App.currentProject.get('scenarios'),
+        }));
+
+        this.listenTo(this.model, 'change:mode', this.showSectionsView);
```

Comment:
Hmm no there isn't. At one point during development it seemed like `modelEvents` was firing too early, so I had to place `listenTo` in `onShow` which is much further down the startup timeline, but it works just as well now so I'll replace it with that.